packages/components/src/FormField/hooks/useFormFieldFocus.ts

```diff
@@ -19,12 +18,6 @@ export function useFormFieldFocus(
   props: UseFormFieldFocusProps,
 ): UseFormFieldFocus {
   const [focused, setFocused] = useState(false);
-  const activeElement = useActiveElement();
-  const activeElementRef = useRef(activeElement);
-
-  useEffect(() => {
-    activeElementRef.current = activeElement;
```

Comment:
previously we were trying to use refs to make sure that we weren't getting a stale version of `activeElement` since our handlers are not getting re-built

however since `document.activeElement` is outside React anyway, we can just use that directly and not have to over complicate things trying to ensure we're not using stale state